zeroconf.c

```diff
@@ -385,9 +385,12 @@ OpcUa_StatusCode OPCUA_DLLCALL ualds_zeroconf_registerInternal(OpcUa_Void*  pvCa
         /* replace [gethostname] in own server name */
         if (bOwnRegistration != OpcUa_False)
         {
-            char hostname[UALDS_CONF_MAX_URI_LENGTH] = {0};
+            // additional 6 chars for '.local'
+            char hostname[UALDS_CONF_MAX_URI_LENGTH+6] = {0};
             ualds_platform_getfqhostname(hostname, sizeof(hostname));
```

Comment:
So the `ualds_platform_getfqhostname` should be changed to include `.local` in the hostname?
Only the hostname itself is not a valid FQDN...